notebooks/multiclass/MulticlassHeartDisease.ipynb

```diff
@@ -286,7 +286,7 @@
    "source": [
     "# Run the scan.\n",
     "# By default this will write the results into individual report files (one per model and evaluation\n",
-    "# type) in the 'local_reports' directory relative to this notebook. This may be disabled by specifying\n",
+    "# type) in the 'reports' directory relative to this notebook. This may be disabled by specifying\n",
```

Comment:
Is the location changed? From local reports to reports

Reply:
@akumar-cs Without any path mentioned the default location would be /reports local to the notebook.

Reply:
Ok